values.yaml

```diff
@@ -198,8 +201,18 @@ prometheus:
       cpu: "2000m"
       memory: "8Gi"
 
+#################################
+## KEDA configuration
+#################################
 
-
+keda:
+  keda:
+    nodeSelector:
+      <<: *platformNodeSelector
+    affinity:
+      <<: *platformAffinity
+    tolerations:
+      *platformTolerations
```

Comment:
Nice workaround

Reply:
@sjmiller609 found the stackOverflow post :).

YAML is magic. Not the good kind.

Reply:
I guess we can use this pattern going forward for any translation to the 3rd party charts. :+1: